markdown_flow/core.py

```
@@ -455,15 +527,25 @@ def _process_content(
+            content = strip_preserve_tags(content)
             return LLMResult(content=content, prompt=messages[-1]["content"])
 
         if mode == ProcessMode.STREAM:
             if not self._llm_provider:
                 raise ValueError(LLM_PROVIDER_REQUIRED_ERROR)
 
             def stream_generator():
+                tag_filter = StreamTagFilter()
                 for chunk in self._llm_provider.stream(messages, model=self._model, temperature=self._temperature):  # type: ignore[attr-defined]
-                    yield LLMResult(content=chunk, prompt=messages[-1]["content"])
+                    # 过滤 preserve_or_translate 标签（处理跨 chunk 分割）
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)